src/backend/base/langflow/api/v1/jobs.py

```diff
@@ -0,0 +1,175 @@
+from __future__ import annotations
+
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, HTTPException
+from loguru import logger
+from pydantic import BaseModel, Field
+from sqlmodel import col, select
+
+from langflow.api.utils import CurrentActiveUser
+from langflow.api.v1.endpoints import simple_run_flow_task
+from langflow.api.v1.schemas import SimplifiedAPIRequest
+from langflow.helpers.flow import get_flow_by_id_or_endpoint_name
+from langflow.services.database.models.flow import Flow
+from langflow.services.database.models.job.model import Job, JobRead, JobStatus
+from langflow.services.deps import get_jobs_service, session_scope
+from langflow.services.jobs.service import JobsService
+
+router = APIRouter(prefix="/jobs", tags=["Jobs"])
+
+
+class CreateJobRequest(BaseModel):
+    """Request model for creating a job.
+
+    Attributes:
+        name (Optional[str]): An optional user-defined name for the job.
+        input_request (SimplifiedAPIRequest): The payload containing all necessary data
+            to run the specified flow. This includes input parameters, stream flags, and other
+            relevant options.
+    """
+
+    name: str | None = None
+    input_request: SimplifiedAPIRequest = Field(..., description="Input request for the flow")
+
+
+@router.post("/{flow_id_or_name}", response_model=str)
+async def create_job(
+    request: CreateJobRequest,
+    user: CurrentActiveUser,
+    flow: Annotated[Flow, Depends(get_flow_by_id_or_endpoint_name)],
+) -> str:
+    """Create a new job for a given flow.
+
+    This endpoint schedules a background job to run the specified flow, using
+    the provided input data and job name. The newly created job will be stored
+    in the job repository and processed asynchronously.
+
+    Args:
+        request (CreateJobRequest): The job creation request parameters, including
```

Comment:
NIT: Typing the arg in the docstring is not really useful since we have it in the signature.